README.md

```diff
@@ -38,12 +38,23 @@ Name | Description
 ## Installation and Usage
 
 ### Installation
-Clone the collection repository.
 
-```shell
-  mkdir -p ~/.ansible/collections/ansible_collections/cloud/azure_ops
-  cd ~/.ansible/collections/ansible_collections/cloud/azure_ops
-  git clone https://github.com/redhat-cop/cloud.azure_ops .
```

Comment:
What if the user does not use the collection from AH?

Reply:
@alinabuzachis  Plz refer to the conversation [here](https://github.com/redhat-cop/cloud.aws_troubleshooting/pull/26#discussion_r1308878223)